src/client/l10n/locales.ts

```diff
@@ -46,6 +46,7 @@ export type LocaleData = {
   }
   callCenter?: CallCenterData
   currencyCode: 'SEK' | 'NOK' | 'DKK'
+  market: 'SWEDEN' | 'NORWAY' | 'DENMARK'
```

Comment:
This look identical to `apiMarket` - is there a difference?

Reply:
no. I just did not notice we have that. will revert.